### PR TITLE
How to set up knife

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

Adds basic setup information to knife_setup.md. Formatting fixes on config_rb.md

## Description
We should tell people how configure knife.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

Aha! Link: https://chef.aha.io/features/SH-2017

### CODE REVIEW

File: www/content/workstation/knife_setup.md [hunk 1]
```diff
@@ -14,81 +14,49 @@ aliases = ["/knife_setup.html", "/knife_setup/"]
...
 
-When running Microsoft Windows, the config.rb file is located at
-`%HOMEDRIVE%:%HOMEPATH%\.chef` (e.g. `c:\Users\<username>\.chef`). If
-this path needs to be scripted, use `%USERPROFILE%\chef-repo\.chef`.
+Previous Chef Infra setups recommended setting up knife with a `config.rb` file. Configuring knife with `config.rb` is still valid if you work on one Chef Infra Server with one Chef Infra Server organization.
 
-{{< /note >}}
+```bash
+mkdir ~/.chef
+touch ~/.chef/config.rb
```
Comment (kagarmoe) [inline www/content/workstation/knife_setup.md:39]: Does knife configure create the file?
Would it also create the `.chef` directory--or does it create the file in its current directory?
```suggestion
knife configure
```

File: www/content/workstation/knife_setup.md [hunk 2]
```diff
@@ -14,81 +14,49 @@ aliases = ["/knife_setup.html", "/knife_setup/"]
...
+```
 
-### config.rb Configuration Within a Chef Repository
+```powershell
+New-Item -Path "c:\" -Name ".chef" -ItemType "directory"
+New-Item -ItemType "file" -Path "c:\.chef\config.rb"
```
Comment (kagarmoe) [inline www/content/workstation/knife_setup.md:44]:
```suggestion
knife configure
```

File: www/content/workstation/knife_setup.md [hunk 3]
```diff
@@ -14,81 +14,49 @@ aliases = ["/knife_setup.html", "/knife_setup/"]
...
-{{< note >}}
+```powershell
+New-Item -Path "c:\" -Name ".chef" -ItemType "directory"
+New-Item -ItemType "file" -Path "c:\.chef\credentials"
+```
 
-When running Microsoft Windows, the config.rb file is located at
-`%HOMEDRIVE%:%HOMEPATH%\.chef` (e.g. `c:\Users\<username>\.chef`). If
-this path needs to be scripted, use `%USERPROFILE%\chef-repo\.chef`.
+Previous Chef Infra setups recommended setting up knife with a `config.rb` file. Configuring knife with `config.rb` is still valid if you work on one Chef Infra Server with one Chef Infra Server organization.
```
Comment (kagarmoe) [inline www/content/workstation/knife_setup.md:35]:
```suggestion
Previous Chef Infra setups recommended setting up knife with a `config.rb` file. Configuring knife with `config.rb` is still valid, but only for working on one Chef Infra Server with one Chef Infra Server organization.
```

File: www/content/workstation/knife_setup.md (hunk 1 above)
Comment (clintoncwolfe) [reply www/content/workstation/knife_setup.md:39]: It appears that `knife configure` will create the `.chef` directory if needed. However, far from creating the `config.rb`, it in fact will complain if it is not present; and then goes on to generate a `credentials` file for managing profiles. I don't think it's the right command for here.

File: www/content/workstation/knife_setup.md (hunk 2 above)
Comment (clintoncwolfe) [reply www/content/workstation/knife_setup.md:44]: See previous comment; `knife configure` is probably not what you want.

File: www/content/workstation/knife_setup.md (hunk 1 above)
Comment (kagarmoe) [reply www/content/workstation/knife_setup.md:39]: But it is the right command for setting up knife to use profiles, which I'm declaring the preferred method. So, to set up knife for profiles (but not for setting up config.rb), do I say "From your root directory, for example `c:\` or `~/` run `knife profile`?

File: www/content/workstation/config_rb.md [hunk 4]
```diff
@@ -28,16 +28,16 @@ A `config.rb` file is used to specify configuration details for knife.
...
--   Is loaded every time the knife executable is run
--   The default location at which Chef Infra Client expects to find this
+- Is loaded every time the knife executable is run
+- The default location at which Chef Infra Client expects to find this
     file is `~/.chef/config.rb`; use the `--config` option from the
     command line to change this location
--   Is not created by default
--   Is located by default at `~/.chef/config.rb` (UNIX and Linux
+- Is not created by default
+- Is located by default at `~/.chef/config.rb` (UNIX and Linux
```
Comment (tas50) [inline www/content/workstation/config_rb.md:36]:
```suggestion
- Is located by default at `~/.chef/config.rb` (macOS and Linux
```

Make's it more clear for macOS users. We don't support BSD or Solaris on Workstation so this is fine.

File: www/content/workstation/config_rb.md [hunk 5]
```diff
@@ -137,51 +107,39 @@ This configuration file has the following settings:
...
+    - CentOS
+    - Windows
 
     Support for FIPS was introduced in Chef Server version 12.13. The
     following operating systems are supported:
 
-    -   Red Hat Enterprise Linux
-    -   Oracle Enterprise Linux
-    -   CentOS
+    - Red Hat Enterprise Linux
```
Comment (tas50) [inline www/content/workstation/config_rb.md:129]: All these OS releases can go away. We're talking about the config.rb so it doesn't matter what platforms you can install Chef Infra Server on.

Comment (kagarmoe) [inline www/content/workstation/config_rb.md:129]:
```suggestion
```

File: www/content/workstation/config_rb.md [hunk 6]
```diff
@@ -137,51 +107,39 @@ This configuration file has the following settings:
...
+    - Oracle Enterprise Linux
+    - CentOS
```
Comment (kagarmoe) [inline www/content/workstation/config_rb.md:131]:
```suggestion
```

File: www/content/workstation/config_rb.md [hunk 7]
```diff
@@ -137,51 +107,39 @@ This configuration file has the following settings:
...
+    - Windows
 
     Support for FIPS was introduced in Chef Server version 12.13. The
     following operating systems are supported:
 
-    -   Red Hat Enterprise Linux
-    -   Oracle Enterprise Linux
-    -   CentOS
+    - Red Hat Enterprise Linux
+    - Oracle Enterprise Linux
```
Comment (kagarmoe) [inline www/content/workstation/config_rb.md:130]:
```suggestion
```

**Pull request merged**